packages/convertkit/src/forms/subscribe/subscribe-to-convertkit-form.tsx

```diff
@@ -20,14 +21,14 @@ type SubscribeFormProps = {
   submitButtonElem?: React.ReactElement
   onError?: (error?: any) => void
   onSuccess?: (subscriber?: ConvertkitSubscriber) => void
-  formId: number
+  formId?: number
```

Comment:
this is optional since we default to `process.env.NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM` later on